qbft/future_msg.go

```diff
@@ -5,19 +5,6 @@ import (
 	"github.com/pkg/errors"
 )
 
```

Comment:
why not remove the entire file?

Reply:
Currently the remaining function is needed so that logic in `msg_validation.go` won't error.

Once @MatheusFranco99 will declare that Msg Validation he was working on is completed, we can delete it in another PR.

Reply:
what msg_validation.go file?

Reply:
https://github.com/bloxapp/ssv-spec/blob/main/p2p/validation/msg_validation.go

Reply:
then lets move it there and remove the file... no need for non used code in spec

Reply:
fixed